### PR TITLE
docs: add bigquery docs

### PR DESCRIPTION
**Pull request by lucasmarshall**
Other docs fixes

### CODE REVIEW

File: openapi/v2/mgmt/components/schemas/create_update_destination.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ allOf:
           - type
           - config
       - type: object
-        title: postgres
+        title: bigquery
```
Comment (albertyfwu) [inline openapi/v2/mgmt/components/schemas/create_update_destination.yaml:35]: nice catch

File: docs/docs/destinations/bigquery.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Pattern #3 - Materialized Views/ETL pipeline
+
+If none of the above patterns solve your use cases, you can build your ETL pipeline transformations and optimize data for your application's read use cases. Use Supaglue's [webhook sync notifications](/api/v2/mgmt/webhooks) to trigger your pipelines.
+
+## Schema Evolution
+
+Supaglue may evolve the destination table schemas from time to time. To evolve schemas, drop your destination tables, and Supaglue will recreate the tables with the new schemas. Please reach out to ([support@supaglue.com](mailto:support@supaglue.com)) if you need support for backward-compatible strategies.
+
+## IP Whitelist
```
Comment (albertyfwu) [inline docs/docs/destinations/bigquery.md:58]: should we put this in a central place instead?

**Review by albertyfwu — APPROVED** [review]

File: docs/docs/destinations/bigquery.md (hunk 2 above)
Comment (lucasmarshall) [reply docs/docs/destinations/bigquery.md:58]: Could be worth doing. I'll defer to @george-xing

**Comment by tomkit** [comment]
Can we provide guidance on grants?

File: docs/docs/destinations/bigquery.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+![postgres-config](/img/bigquery-form.png)
+
+## Query patterns
+
+Here are a few high-level best practices when working with tables that Supaglue land:
+
+- Avoid altering the schema of the existing columns
+
+There are a few patterns (from simplest to more complex) for querying tables that Supaglue writes into your Postgres:
```
Comment (george-xing) [inline docs/docs/destinations/bigquery.md:40]: @lucasmarshall fix references to postgres?

Comment (lucasmarshall) [reply docs/docs/destinations/bigquery.md:40]: Good catch

**Pull request merged**